frontend/model/state.js

```diff
@@ -157,6 +157,28 @@ sbp('sbp/selectors/register', {
           upgradeAction(contractID)
         })
       }
+    })();
+    (() => {
+      // Update expired invites
+      // If fewer than MAX_GROUP_MEMBER_COUNT 'anyone can join' have been used,
+      // create a new 'anyone can join' link up to MAX_GROUP_MEMBER_COUNT invites
```

Comment:
Can this comment be updated to add that we're not just creating a new invite, but we're replacing the old one?

Reply:
I don't think it's inaccurate. Yes, we're creating a new one, but the purpose is to replace the old one, and this isn't mentioned in the comment.

Reply:
https://github.com/okTurtles/group-income/pull/2366#discussion_r1790653189